netmaster/objApi/apiController.go

```diff
@@ -667,8 +670,12 @@ func (ac *APIController) EndpointGroupDelete(endpointGroup *contivModel.Endpoint
 			endpointGroup.GroupName, endpointGroup.Links.AppProfile.ObjKey)
 	}
 
-	endpointGroupCleanup(endpointGroup)
-	return nil
+	err := endpointGroupCleanup(endpointGroup)
+	if err != nil {
+		log.Errorf("EPG cleanup failed: %+v", err)
```

Comment:
The previous return will be printed here. So, it should contain the EPG name as well